proto/lighthouse-result.proto

```diff
@@ -161,6 +161,9 @@ message LighthouseResult {
       string entry_type = 2;
       google.protobuf.DoubleValue start_time = 3;
       google.protobuf.DoubleValue duration = 4;
+
+      // Whether timing entry was collected during artifact gathering.
+      bool gather = 5;
```

Comment:
didn't @exterkamp *just* remove this ? maybe I'm just misremembering what I saw haha

Reply:
Ha, unless he had amnesia I don't think so... @exterkamp?

Reply:
I removed it from my PR so it could be here where it made more sense 😄